lib/web/apiserver.go

```diff
@@ -167,6 +167,10 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*RewritingHandler, error) {
 	h.PUT("/webapi/users/password", h.WithAuth(h.changePassword))
 	h.POST("/webapi/sites/:site/namespaces/:namespace/users/password/token", h.WithClusterAuth(h.createResetPasswordToken))
 
+	// User CRUDS that a user in context can apply to other users.
+	h.POST("/webapi/sites/:site/namespaces/:namespace/users", h.WithClusterAuth(h.createUser))
```

Comment:
```
h.POST("/webapi/sites/:site/namespaces/:namespace/users", h.WithClusterAuth(h.createUser))
h.GET("/webapi/sites/:site/namespaces/:namespace/users", h.WithClusterAuth(h.getUsers))
```

we will not be supporting managing users on remote clusters so:

```
"/webapi/users"
```

Reply:
as discussed on the call, i kept it the same, with the frontend defaulting `clusterId` to root's cluster